Backend/Tools/astrosScraper.py

```diff
@@ -0,0 +1,71 @@
+import json
+import requests
+from Backend.Requests import astrosOnISS
+from Backend.Tools.AstrosJsonLoader import process
+
+def getAstroObject(name):
+    # get image url
+    headers = {
+        "apikey": "hereWriteAPIKey"}
```

Comment:
here we have to enter the apiKey. Doesn't make sense to write this here of course. But now idea how to solve this.

Reply:
The correct way to solve this is by creating an environment variable which holds the api key and reading it in the script. It's pretty easy to do. In Python it's one command and outside of python it depends on your os. Linux it's 1 shell command, windows is clicking through uis and setting it. So you shouldn't habe trouble implementing it. But get in touch with jojo, because the vatriable has to be set in the docker aswell.